src/router/utils/HelperFunctions.jsx

```diff
@@ -178,13 +156,10 @@ export const clearHistory = async (videoDispatch) => {
         type: "CLEAR_HISTORY",
         payload: response.data.history,
       });
-      Toast({ type: "success", message: "Cleared History" });
+      toast.success("Cleared History");
     }
   } catch (error) {
-    Toast({
-      type: "error",
-      message: "Oops! An error occurred.Try again later.",
-    });
+    toast.error("Oops! An error occurred.Try again later.");
     console.log(error.message);
```

Comment:
remove all the console.log

Reply:
Thank you. These have been kept intentionally.